parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/xcm_config.rs

```diff
@@ -158,7 +198,7 @@ pub type LocalOriginToLocation = SignedToAccountId32<RuntimeOrigin, AccountId, R
 /// queues.
 pub type XcmRouter = (
 	// Two routers - use UMP to communicate with the relay chain:
-	cumulus_primitives_utility::ParentAsUmp<ParachainSystem, PolkadotXcm>,
+	cumulus_primitives_utility::ParentAsUmp<ParachainSystem, PolkadotXcm>, // TODO: Figure out 3rd type arg
```

Comment:
@bkontur This and the corresponding one in Rococo bridge hub will need your help in fixing. The 3rd argument here details the price of sending an XCM to the parent, which I believe is the relay chain in both Kusama and Rococo.

Reply:
@KiChjang 
at the begining, I would add there `ParentAsUmp<ParachainSystem, PolkadotXcm, ()>` as the other runtimes have.

we have TODO issue for final setup of fees (https://github.com/paritytech/parity-bridges-common/issues/1677), so we will come back later to this